src/web/app/page.component.scss

```diff
@@ -14,10 +14,22 @@ nav {
 
 .main-content {
   margin-top: 50px;
-  min-height: calc(100vh - 80px);
+  min-height: calc(100vh - 102px);
```

Comment:
this update was after master branch is merged, I was wondering why is this still shown as a difference with master branch?

Reply:
I believe the issue is because GitHub does not rebase the commit history automatically in pull requests.

[Good stackoverflow article](https://stackoverflow.com/questions/16306012/github-pull-request-showing-commits-that-are-already-in-target-branch) that resolves this issue :)